### PR TITLE
Optimum Nvidia Mistral, Memory support, qualitative comparision and improvements

### PR DESCRIPTION
**Pull request by Anindyadeep**
This PR introduces all the changes by PR https://github.com/premAI-io/benchmarks/pull/167 and integrates those in Optimum Nvidia. Optimum Nvidia README now has quality checks table for both Llama 2 Chat and Mistral Instruct.

Note this PR depends on: #175 

So first PR #175 needs to be merged then this one

### CODE REVIEW

File: bench_exllamav2/bench.sh
Comment (nsosio) [inline bench_exllamav2/bench.sh:1]: wrong file?

**Review by nsosio — CHANGES_REQUESTED** [review]
minor things related to exllaamv2

File: bench_exllamav2/bench.sh
Comment (Anindyadeep) [reply bench_exllamav2/bench.sh:1]: Yes, sorry it was by mistake, fixed it

File: docs/llama2.md.template [hunk 1]
```diff
@@ -18,6 +18,8 @@
 | [ctransformers](/bench_ctransformers/)      |      -       |      -         | 85.50 ± 1.00  | 86.66 ± 1.06  |
 | [llama.cpp](/bench_llamacpp/)               |      -       |      -         | 89.90 ± 2.26  | 97.35 ± 4.71  |
 | [ctranslate](/bench_ctranslate/)            | 46.26 ± 1.59 | 79.41 ± 0.37   | 48.20 ± 0.14  |      -        |
+| [exllamav2](/bench_exllamav2/)              |      -       |      -         | 125.58 ± 1.23 | 159.68 ± 1.85 |
```
[the remark this answers is not in the document]
Comment (Anindyadeep) [reply docs/llama2.md.template]: I just merged the changes with dev and now it should hopefully be fine

**Review by nsosio — APPROVED** [review]
LGTM

**Pull request merged**